src/pages/index.html

```diff
@@ -37,7 +37,8 @@
 			<div class="section mod_slider color_lightblue">
 				<div class="section__content">
 					<div class="slider">
-						<a href="https://sap.github.io/ui5-webcomponents" target="_blank" rel="noopener" class="slider__item"></a>
+						<a href="/ui5con/" class="slider__item mod_ui5con"></a>
```

Comment:
The first link is opened in a separate tab. The second one - in the same.
Should not the behavior be consistent?

Reply:
Well, original idea was to open same origin links in the same window and foreign origin in new window. But you are right, let's keep it consistent and open all promotions in new window.